build_tools/cmake/iree_llvm.cmake

```diff
@@ -150,7 +150,6 @@ macro(iree_llvm_set_bundled_cmake_options)
 
   # Unconditionally enable some other cheap LLVM tooling.
   set(IREE_LLVM_LINK_TARGET llvm-link)
-  set(IREE_LLD_TARGET lld)
```

Comment:
Sorry about the fly-by comment.

There are other `IREE_LLD_TARGET` usages in this file (L75, L108, L168, L174) that still point to `lld`. Do we expect to clean those too?

Reply:
I think those other uses are still correct. The change on this line makes it so the `lld` target (part of LLVM) is not depended on directly unless building the LLVM_CPU target backend.

Reply:
Agreed with what Scott says.